content/4_Gold/SP.mdx

```diff
@@ -358,13 +358,11 @@ public class test {
 
 <Warning>
 
-It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem!
+It's important to include `continue`. This ensures that when all edge weights are non-negative, we will never go through the adjacency list of any vertex more than once. Removing it will cause TLE on the last test of the above problem! 
 
-<IncompleteSection>
+The last testcase contains 100000 destinations and 149997 flights. City 1 has around 50000 flights to cities 2 through 50000. Cities 2 through 50000 have flights that go to city 50001. City 50001 has flights that go to cities 50002 through 100000. Without the `continue`, after the program pops cities 1 through 50000 off the queue, the priority queue will contain 49999 routes that end at city 50001. After getting the shortest cost to city 50001 from these flights, it will move on to the cities 50002 through 100000 that have flights from city 50001. This means that city 50001 will be processed 49998 more times than needed (deleted the part but maybe i should rephrase it?). This results in a runtime of $\Theta(N^2\log N)$, which will TLE. 
```

Comment:
remove "around 50000"

Reply:
use "flights to" rather than "flights that go to"

Reply:
> After getting the shortest cost to city 50001 from these flights

routes, not flights

Reply:
> This means that city 50001 will be processed 49998 more times than needed (deleted the part but maybe i should rephrase it?).

What about - Every one of the 49999 times city 50001 is popped off the queue and processed, we must iterate over all of its outgoing flights (cities 50002 - 10^5).